dev/io.openliberty.org.eclipse.microprofile/config.3.1.bnd

```diff
@@ -0,0 +1,34 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.org.eclipse.microprofile.config.3.1; singleton:=true
+
+Import-Package: \
+  *
+
+Export-Package: \
+  org.eclipse.microprofile.config;version=3.0, \
+  org.eclipse.microprofile.config.inject;version=2.0, \
+  org.eclipse.microprofile.config.spi;version=3.0
+
+Include-Resource: \
+  @${repo;org.eclipse.microprofile.config:microprofile-config-api;3.0.2;EXACT}
+
+-maven-dependencies: \
+   dep1;groupId=org.eclipse.microprofile.config;artifactId=microprofile-config-api;version=3.0.2;scope=runtime
```

Comment:
This modifies the the pom.xml which is generated within the bundle to add a dependency on the upstream microprofile-config-api with `runtime` scope.

Why do we do this? It seems a bit odd but it's consistent with the other MP API bundles in this project.

Reply:
I don't know ... I think @jhanders34 added this in the previous version

Reply:
Chuck added this for the mend stuff to know which version is depended on.  With doing sub-bundles it doesn't work the same way.

Reply:
It is just used for reporting during the build and then stripped from the bundle's pom file when done.

Reply:
Good to know, thanks :+1: